src/lib/services/feature-toggle-service.ts

```diff
@@ -704,7 +710,7 @@ class FeatureToggleService {
         userId?: number,
         archived: boolean = false,
     ): Promise<FeatureToggle[]> {
-        return this.featureToggleClientStore.getAdmin({
+        return this.featureToggleAdminStore.getAdmin({
```

Comment:
admin is returning not just FeatureToggle but also the client stuff that some tests rely on